docs/source/esp-idf_component.rst

```diff
@@ -37,6 +37,8 @@ Installation
     cd ../.. && \
     idf.py menuconfig
 
+.. note:: If you use Arduino with IDF often, you can place the arduino folder into global components folder.
```

Comment:
ESP-IDF instead of just "IDF"?

Reply:
OK, I was waiting if there was anything else to be changed... pushed the change...